src/Telemetry/Samples/FSharp/FSharp.fsproj

```diff
@@ -7,7 +7,7 @@
     <RootNamespace>Microsoft.Quantum.Telemetry.Samples.FSharp</RootNamespace>
     <!--
     The MsBuild imports cause a bunch of MSB3277 errors that seem benign.
-    I believe the root is the use of the CommandLineParser that seems to target netstandard2.0
+    I believe the root is the use of the CommandLineParser that seems to target net6.0

```

Comment:
Looks like a comment that got swept up in the Find+Replace?

Reply:
Agreed, thanks. I reverted the change.